Homework1/Sorting/Program.cs

```diff
@@ -0,0 +1,89 @@
+﻿namespace Sorting
+{
+    internal class Program
+    {
+        private static int Main(string[] args)
+        {
+            if (!AreTestsCorrect())
+            {
+                Console.WriteLine("Tests failed");
+                return -1;
+            }
+
+            Console.Write("Enter elements of array: ");
+            var sequenceOfNumbers = Console.ReadLine();
+            if (sequenceOfNumbers == null)
+            {
+                Console.WriteLine("No elements for sorting");
+                return 0;
+            }
+
+            int[] arrayOfNumbers = MakeArrayFromString(sequenceOfNumbers);
+            SortByInsertion(arrayOfNumbers);
+            Console.Write("Sorted array: ");
+            for (var i = 0; i < arrayOfNumbers.Length; i++)
+            {
+                Console.Write(arrayOfNumbers[i] + " ");
+            }
+
+            Console.WriteLine();
+            return 0;
+        }
+
+        private static int[] MakeArrayFromString(string sequenceOfNumbers)
+        {
+            string[] stringArrayOfNumbers = sequenceOfNumbers.Split(' ');
+            int[] intArrayOfNumbers = new int[stringArrayOfNumbers.Length];
```

Comment:
```suggestion
            var intArrayOfNumbers = new int[stringArrayOfNumbers.Length];
```
 В современном C# почти никогда не надо писать тип в одной строке дважды